### PR TITLE
Adding accounts

### PR DESCRIPTION
**Pull request by willi123yao**
Basic design of login.php and registration.php
Fixed title to match current project title

##
Tell me on discord if any mistakes or here.
For merging, use squash

### CODE REVIEW

File: user/login.php [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+?>
+
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="../css/materialize.css">
+  <script type="text/javascript" src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+  <script type="text/javascript" src="../js/materialize.min.js"></script>
+  <title>Login - NyaaPantsu</title>
```
Comment (gensuihime) [inline user/login.php]: Should be Re:Nyaa cuz nyaapantsu isn't our proj name

File: view.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
   <link rel="stylesheet" href="css/materialize.css">
   <script type="text/javascript" src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
   <script type="text/javascript" src="js/materialize.min.js"></script>
-  <title>Home - AniLove</title>
+  <title>View - ReNyaa</title>
```
Comment (alexhgian) [inline view.php]: Can you make it `Re:Nyaa` instead of `ReNyaa`

File: view.php [hunk 3]
```diff
@@ -164,7 +164,7 @@ function clicksearch(){
       <ul class="hide-on-med-and-down">
         <li><a onclick="showmenu()"><i class="material-icons">menu</i></a></li>
       </ul>
-        <a href="#" class="brand-logo">AniLove</a>
+        <a href="#" class="brand-logo">ReNyaa</a>
```
Comment (alexhgian) [inline view.php]: Similar to above.

File: index.php [hunk 4]
```diff
@@ -5,7 +5,7 @@
   <link rel="stylesheet" href="css/materialize.css">
   <script type="text/javascript" src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
   <script type="text/javascript" src="js/materialize.min.js"></script>
-  <title>Home - AniLove</title>
+  <title>Home - ReNyaa</title>
```
Comment (alexhgian) [inline index.php]: Can you change to `Re:Nyaa`

File: index.php [hunk 5]
```diff
@@ -155,7 +155,7 @@ function clicksearch(){
       <ul class="hide-on-med-and-down">
         <li><a onclick="showmenu()"><i class="material-icons">menu</i></a></li>
       </ul>
-        <a href="#" class="brand-logo">AniLove</a>
+        <a href="#" class="brand-logo">ReNyaa</a>
```
Comment (alexhgian) [inline index.php]: Same as above.

File: view.php (hunk 2 above)
Comment (alexhgian) [inline view.php]: Think of Re:Zero

**Pull request merged**